ioctl.go

```diff
@@ -0,0 +1,60 @@
+package gpio
```

Comment:
Please make everything in this file not exported, like `func Io -> io` because it's not part of library API.